feat(helm): make nginx resolver configurable (#9491)
grafana/loki: docs/sources/installation/helm/reference.md

			<td><pre lang="json">
"main '$remote_addr - $remote_user [$time_local]  $status '\n        '\"$request\" $body_bytes_sent \"$http_referer\" '\n        '\"$http_user_agent\" \"$http_x_forwarded_for\"';"
</pre>
</td>
		</tr>
		<tr>
			<td>gateway.nginxConfig.resolver</td>
			<td>string</td>

> 			<td>Allows to set a custom resolver</td>

Reviewer: ```suggestion
			<td>Lets you set a custom resolver</td>
```
